### PR TITLE
:ambulance: saas_portal_portal: fix ``/my/home`` views to comply v11,…

### PR DESCRIPTION
**Pull request by iledarn**
… fix database checking js

### CODE REVIEW

File: saas_portal_start/static/src/js/main.js [hunk 1]
```diff
@@ -36,7 +36,6 @@ $(document).ready(function() {
             $input.attr('data-content', "Please choose your domain name");
             error = true;
         }
-        db_name = _.str.slugify(db_name);
```
Comment (yelizariev) [inline saas_portal_start/static/src/js/main.js:39]: Why?

Comment (iledarn) [reply saas_portal_start/static/src/js/main.js:39]: because it removes dots

Comment (yelizariev) [reply saas_portal_start/static/src/js/main.js:39]: Using dots change the idea of module (quick start on a sub domain). It should be optional or in another module

Comment (iledarn) [reply saas_portal_start/static/src/js/main.js:39]: Ok, I can put back this ``slugify`` thing. But for me, it is not friendly user interface just to silently replace dots that a user for some reason wants to be there

Comment (iledarn) [reply saas_portal_start/static/src/js/main.js:39]: I don't see either how dots change the idea of this module

Comment (iledarn) [reply saas_portal_start/static/src/js/main.js:39]: A user can quickly start on a subdomain regardless of any dots in the input field. With dots or without dots - it doesn't matter at all

**Comment by iledarn** [comment]
superseded by https://github.com/it-projects-llc/odoo-saas-tools/pull/800 (the same, but without removing the slugify)

File: saas_portal_start/static/src/js/main.js (hunk 1 above)
Comment (yelizariev) [reply saas_portal_start/static/src/js/main.js:39]: Slugify is not for dots removing. It changes any non ascii symbols

Comment (iledarn) [reply saas_portal_start/static/src/js/main.js:39]: It removes dots. Nonascii symbols are detected by regex anyway

**Pull request closed without merging**